rskj-core/src/main/java/org/ethereum/vm/program/Program.java

```diff
@@ -1608,8 +1608,12 @@ public static RuntimeException tooLargeContractSize(@Nonnull Program program, in
             return new RuntimeException(format("Maximum contract size allowed %d but actual %d, tx: %s", maxSize, actualSize, extractTxHash(program)));
         }
 
+        public static RuntimeException tooLargeInitCodeSize(@Nonnull Program program, int maxSize, int actualSize) {
+            return new RuntimeException(format("Maximum initcode size allowed %d but actual was %d, tx: %s", maxSize, actualSize, extractTxHash(program)));
+        }
+
         public static AddressCollisionException addressCollisionException(@Nonnull Program program, RskAddress address) {
-            return new AddressCollisionException("Trying to create a contract with existing contract address: 0x" + address + ", tx: " + extractTxHash(program));
+            return new AddressCollisionException("Tried to create a contract with an already existing contract address: 0x" + address + ", tx: " + extractTxHash(program));
```

Comment:
as this part is existing and sort of unrelated, I wouldn't change it unless it's really necessary. That wording might be in use by some toolings/parsers

Reply:
Got it. Didn't know that. This was a suggestion added by @nagarev . Will remove then